pandas/core/generic.py

```diff
@@ -5138,9 +5139,11 @@ def describe_1d(data):
         if self.ndim == 1:
             return describe_1d(self)
         elif (include is None) and (exclude is None):
-            if len(self._get_numeric_data()._info_axis) > 0:
+            if len(self._get_numeric_data(exclude_bool=True)._info_axis) > 0:
                 # when some numerics are found, keep only numerics
                 data = self.select_dtypes(include=[np.number])
```

Comment:
is there any reason to change ``_get_numeric_data`` rather than changing this line?